lib/spatial_features/importers/kml.rb

```diff
@@ -106,6 +114,13 @@ def extract_table(placemark)
         end
         return metadata
       end
+
+      def strip_altitude(kml)
+        kml.css('coordinates').each do |coordinates|
+          next unless COORDINATES_WITH_ALTITUDE.match?(coordinates.content)
```

Comment:
`match?` before `gsub` to short-circuit a potentially expensive regex replacement on thousands of coordinate tuples